README.md

```diff
@@ -1,2 +1,75 @@
-# mint-signup-page
-Mint Sign Up Page - Microverse Collaborative Project Between Stanley and Umair
+
+# mint-signup-page (Embedding Images and Video)
+
+> Microverse Collaborative Project (Stanley and Umair)
+
+![screenshot](img/signup.PNG)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML,
+- CSS,
+
+## Live Demo
+
+[Live Demo Link](https://github.com/happiguru/mint-signup-page/)
```

Comment:
- Add a working live demo link.

Reply:
Hi Tirthajyoti
I added the live demo link. Please review it